src/index.ts

```diff
@@ -89,16 +90,22 @@ export class HTTPSnippet {
 
     // is it har?
     if (isHarEntry(input)) {
-      entries = input.log.entries;
+      this.entries = input.log.entries;
     } else {
-      entries = [
+      this.entries = [
         {
           request: input,
         },
       ];
     }
+  }
 
-    entries.forEach(({ request }) => {
+  async init() {
```

Comment:
Any reason why we have this as a separate function vs making .convert() the async function? Is it because this only gets called once? We could perhaps make .convert() async instead, then internally call .init() the first time convert is called? 

Just a passing thought... this comment can safely be ignored.

Reply:
same question, the constructor _and_ the init function being required seems like a smell to me

Reply:
Refactored this to have `convert` be async and interall call `this.init()` if it hasn't yet been.